service/RoleBandService.ts

```diff
@@ -0,0 +1,16 @@
+import { RoleBandResponse } from "../model/RoleBandResponse";
+const axios = require('axios');
+
+axios.defaults.baseURL = process.env.API_URL;
+
+module.exports.URL='http://localhost:8080/api/role-band-levels';
+
+module.exports.getRoleBands = async function (): Promise<RoleBandResponse> {
+    try {
+        const response = await axios.get(this.URL)
```

Comment:
I think only the base url should be an environment variable. Does the name here match what @martin-doherty1 has used in his PR?

```suggestion
        const response = await axios.get(`${this.baseURL}/api/role-band-levels`)
```